Dockerfile.api

```diff
@@ -0,0 +1,9 @@
+FROM node:alpine
+
+WORKDIR /opt/pizza-sync/backend
+ADD ./backend .
+RUN yarn
+
+EXPOSE 3000
```

Comment:
I am not really fan of exposing the project on port 3000 in the Dockerfile. I think you should expose it on port 80. This will be useful in the future if we want to run it on kubernetes.